great_ape_safe/ape_api/helpers/balancer/stable_math.py

```diff
@@ -0,0 +1,337 @@
+# https://github.com/balancer-labs/balpy/blob/a907f7b984f4e3ba3460a1ef064003d95da5e884/balpy/balancerv2cad/src/balancerv2cad/StableMath.py
```

Comment:
is there a reason you removed the `getcontext().prec = 28` lines? i believe this is setting the precision in the background for the decimal module, which is definitely being used no?

Reply:
including `getcontext().prec = 28` gives this error: 

`vyper.exceptions.DecimalOverrideException: Overriding decimal precision disabled`

 will have to look more into how much of a difference this is actually making